validations/transfer.test.ts

```diff
@@ -157,17 +188,40 @@ describe('/transfer', () => {
           TransferStatus.TransferStarted,
           TransferStatus.TransferReadyForUserToSendCryptoFunds,
         ])
-
-	const transferStatusResponse =
+        const transferId = transferOutResponse.unwrap().transferId
+        const transferStatusResponse =
           await fiatConnectClient.getTransferStatus({
-            transferId: transferOutResponse.unwrap().transferId,
+            transferId,
           })
         expect(transferStatusResponse.isOk).to.be.true
         await checkObjectAgainstModel(
           transferStatusResponse.unwrap(),
           'TransferStatusResponse',
         )
 
+        // Webhook validation
+        if (config.clientApiKey && config.providerId) {
```

Comment:
same here (separate test)

Reply:
also it looks nearly identical to the transfer out in case. probly a good opportunity for `it.each`